include/clap/ext/draft/audio-ports-activation.h

```diff
@@ -39,10 +39,10 @@ typedef struct clap_plugin_audio_ports_activation {
 
    // Activate the given port.
    //
-   // It is only possible to activate on the audio-thread if can_activate_while_processing() returns
-   // true.
+   // It is only possible to activate and de-activate on the audio-thread if
```

Comment:
maybe we can write (de)activate to make it shorter?

Reply:
I literally used the wording from line 7 in the same file.